### PR TITLE
Bug fixes + SPM option to show mask

### PR DESCRIPTION
**Pull request by mohanadarafe**

### CODE REVIEW

File: livingpark_utils/livingpark_utils.py [hunk 1]
```diff
@@ -693,6 +692,9 @@ def export_spm_segmentations(
         self,
         cohort: pd.DataFrame,
         folder: str,
+        mni_space: bool,
+        show_wm: bool,
+        show_gm: bool,
```
Comment (glatard) [inline livingpark_utils/livingpark_utils.py]: please add default values and documentation for these parameters.

File: livingpark_utils/livingpark_utils.py [hunk 2]
```diff
@@ -745,18 +747,20 @@ def export_spm_segmentations(
                 os.path.join(self.data_cache_path, "inputs"),
                 os.path.join("outputs", "pre_processing"),
             )
-            output_file_c1 = output_file_name.replace("PPMI", "smwc1PPMI")
-            output_file_c2 = output_file_name.replace("PPMI", "smwc2PPMI")
+            output_file_c1 = output_file_name.replace("PPMI", "c1PPMI", 1)
+            output_file_c2 = output_file_name.replace("PPMI", "c2PPMI", 1)
```
Comment (glatard) [inline livingpark_utils/livingpark_utils.py]: This should be conditional to the use of mni_space=False

File: livingpark_utils/livingpark_utils.py [hunk 3]
```diff
@@ -745,18 +747,20 @@ def export_spm_segmentations(
...
-                output_file_c2, cmap="Blues", threshold=0.1, alpha=alpha
+                anat_img=input_file, cut_coords=list(cut_coords) if cut_coords else None, figure=fig, title=f"#{i}/{len(cohort)}"
             )
 
+            if mni_space or show_gm:
+                display.add_overlay(output_file_c1, cmap="Reds", threshold=0.1, alpha=alpha)
+            
+            if mni_space or show_wm:
+                display.add_overlay(output_file_c2, cmap="Blues", threshold=0.1, alpha=alpha)
+
```
Comment (glatard) [inline livingpark_utils/livingpark_utils.py:778]: The logic is not right: when showing in MNI space then anat_image is not shown (anat_image is in native space). Or else, use normalized anat_image. In addition, show_gm and show_wm should work wheter in mni_space or in native space.

File: livingpark_utils/livingpark_utils.py [hunk 4]
```diff
@@ -791,7 +795,7 @@ def make_gif(self, frame_folder: str, output_name: str = "animation.gif") -> Non
         )
         print(f"Wrote {os.path.join(frame_folder, output_name)}")
 
-    def qc_spm_segmentations(self, cohort) -> None:
+    def qc_spm_segmentations(self, cohort, mni_space = False, show_gm = False, show_wm = False, cut_coords=(-28, -7, 17)) -> None:
```
Comment (glatard) [inline livingpark_utils/livingpark_utils.py]: default values should be all set to True. With the current default values, nothing is displayed.

**Review by glatard — CHANGES_REQUESTED** [review]
Thanks for doing this @mohanadarafe, see my comments inline. Please also make sure to run `pre-commit run --all` before committing.

**Comment by mohanadarafe** [comment]
HI @glatard thank you for the comments, I have updated the PR accordingly.

**Review by glatard — APPROVED** [review]

**Pull request merged**